truncate_core/src/board.rs

```diff
@@ -270,6 +270,20 @@ impl Board {
         }
     }
 
+    pub fn get_mut<'a>(
```

Comment:
Confused how this didn't already exist, but all g. I guess we used a `set`ter?

Reply:
Yeah we always `set`, or I think reach into the Vecs by hand to modify